README.md

```diff
@@ -126,11 +147,11 @@ To begin benchmarking your first embodied agent with NETT, please be aware:
 
 - **Environment**
 
-   Create the simulation environment using the path to your Unity executable (see Step 1).
+   Create the simulation environment using the path to your Unity executable (see Step 1). The path must be absolute and not relative.
    ```python
-   environment = Environment(config="identityandview", executable_path="path/to/executable.x86_64")
+   environment = Environment(executable_path="path/to/executable.x86_64")
    ```
-   To get a list of all available configurations, run `nett.list_configs()`.
+   *(To get a list of all available configurations, run `nett.list_configs()`. If you have not already imported nett, you will need it to run this command: `import nett`.)*
```

Comment:
Line can be deleted since that function no longer exists.
```suggestion
```

Reply:
Great catch!